front/src/modules/trainschedule/components/ManageTrainSchedule/helpers/__tests__/formatTimetableItemPayload.spec.ts

```diff
@@ -173,86 +174,242 @@ describe('formatTimetableItemPayload', () => {
       },
     },
   };
-  describe('when user updates a paced train', () => {
-    describe('user modified paced train category so that it matches the exception category, and the exception is only a category exception', () => {
-      const userChanges: Record<string, TrainCategory> = {
-        category: 'NIGHT_TRAIN',
+  describe('User creates a paced train', () => {
+    it('should add the exceptions defined by user', () => {
+      const userChanges: Partial<OperationalStudiesConfState> = {
+        addedExceptions: [
+          {
+            key: 'az',
+            startTime: new Date('2025-07-01T00:00:00'),
+          },
+        ],
       };
       const osrdconfWithUserChanges: OperationalStudiesConfState = {
-        ...osrdconf,
-        category: userChanges.category,
+        ...rawOsrdconf,
+        ...userChanges,
       };
-      const timetableItemToEditDataWithOneChangeGroup: TimetableItemToEditData = {
-        ...timetableItemToEditData,
-        originalPacedTrain: {
-          ...timetableItemToEditData.originalPacedTrain,
-          exceptions: [
+      const result = formatPacedTrainPayload(osrdconfWithUserChanges, rollingStockName);
+      expect(result).toEqual({
+        category: 'FREIGHT_TRAIN',
+        comfort: 'STANDARD',
+        constraint_distribution: 'MARECO',
+        initial_speed: 0,
+        labels: [],
+        margins: { boundaries: [], values: ['0%'] },
+        options: {
+          use_electrical_profiles: true,
+          use_speed_limits_for_simulation: true,
+        },
+        path: [
+          {
+            id: '0-0',
+            trigram: 'WS',
+            secondary_code: 'BV',
+            track_reference: null,
+            deleted: false,
+          },
+          {
+            id: '1-1',
+            trigram: 'SS',
+            secondary_code: 'BV',
+            track_reference: null,
+            deleted: false,
+          },
+        ],
+        power_restrictions: [],
+        rolling_stock_name: 'DUAL-MODE_RS_E2Ee',
+        schedule: [],
+        speed_limit_tag: undefined,
+        start_time: '2025-06-02T12:45:00.000Z',
+        train_name: 'test',
+        paced: { time_window: 'PT3H', interval: 'PT1H' },
+        exceptions: [
+          {
+            key: 'az',
+            start_time: {
+              value: '2025-07-01T00:00:00.000Z',
+            },
+          },
+        ],
+      });
+    });
+  });
+  describe('User updates a paced train', () => {
+    describe('User creates 1 more added exceptions, and 2 already existed', () => {
```

Comment:
Should it rather be that ?
```suggestion
    describe('User creates 2 more added exceptions, and 1 already existed', () => {
```

Reply:
yes :)
fixes